wondrous-app/services/apollo.ts

```diff
@@ -30,7 +32,7 @@ const authLink = setContext((_, { headers }) => {
   return {
     headers: {
       ...headers,
-      Authorization: auth,
+      ...(!!auth && { Authorization: auth }),
```

Comment:
would this cause issue else where? i suppose if we consistently use '?.' then we are fine

Reply:
shouldn't be causing any issues really, but probably we don't even need the !!auth check -> will send Authorization: auth even with null auth

Reply:
gotcah nbd either way, although for simplicity of code sake Authorization: auth is better because `...(!!auth && { Authorization: auth }), ` is not immediately obvious